pages/linux/synopkg.md

```diff
@@ -0,0 +1,36 @@
+# synopkg
+
+> Package management utility for Synology DiskStation Manager.
+> More information: <https://www.synology.com/dsm>.
+
+- List installed packages without a description:
+
+`synopkg list --name`
+
+- List packages which depend on a specific package:
+
+`synopkg list --depend-on {{package}}`
+
+- Start/Stop a package:
```

Comment:
Not entirely sure about the capitalization here, but I think it should be like this:
```suggestion
- Start/stop a package:
```
Even tough I'd actually prefer
```suggestion
- Start or stop a package:
```

Reply:
I copied the format used in the `systemctl` page.

Reply:
I wouldn't consider many docs trustworth in terms of correct grammer. @SethFalco what would you say?

Reply:
> I wouldn't consider many docs trustworth in terms of correct grammer.

Agreed! It's always handy to improve docs. Most devs aren't very passionate about technical writing. ^-^'

---

I don't know which is valid between "Start/Stop" vs "Start/stop" to be honest. 🤔
I can't say I specifically have a preference between the 2 suggestions, either.

Probably depends on what the repository prefers:
* Something technical users would be comfortable consuming.
* Formal writing that could be in articles, books, education, etc.

Both are unambiguous, easy to consume, and familiar to users. The top is _probably_ more common for technical readers, while the bottom is negligibly more verbose but more formal, i.e. for articles or education.

TL;DR: I can't provide a strong opinion, but maybe that'd help?

Reply:
I suggest `Start or stop a package:` would be the best option here. If you really want the forward slash, then `Start/Stop` would be my preferred suggestion.